README.md

```diff
@@ -6,60 +6,42 @@ docker/cli
 This repository is the home of the cli used in the Docker CE and
 Docker EE products.
 
-It's composed of 3 main folders
-
-* `/cli` - all the commands code.
-* `/cmd/docker` - the entrypoint of the cli, aka the main.
-
 Development
 ===========
 
-### Build locally
+`docker/cli` is developed using Docker.
 
-```
-$ make build
-```
+Build a linux binary:
 
 ```
-$ make clean
+$ make -f docker.Makefile binary
```

Comment:
Thanks.

I agree `make` is pretty standard for some languages, but I don't see it widely used by many golang projects. If it is a standard it's not very widely adopted. I think there are a lot more issues with make beyond just handling docker images, but Ill leave that for another thread.